src/util/system.cpp

```diff
@@ -547,7 +547,7 @@ void ArgsManager::AddArg(const std::string& name, const std::string& help, const
 
     LOCK(cs_args);
     std::map<std::string, Arg>& arg_map = m_available_args[cat];
-    auto ret = arg_map.emplace(name.substr(0, eq_index), Arg(name.substr(eq_index, name.size() - eq_index), help, debug_only));
+    auto ret = arg_map.emplace(name.substr(0, eq_index), Arg{name.substr(eq_index, name.size() - eq_index), help, debug_only});
```

Comment:
Can't test at the moment but I think you could drop `Arg`?

Reply:
> Can't test at the moment but I think you could drop `Arg`?

It won't work if a type is automatically deduced.

Reply:
I'd prefer to keep it. Three ascii chars are not a bloat